examples/run/alpaka/full_chain_algorithm.hpp

```diff
@@ -122,9 +118,9 @@ class full_chain_algorithm
 
     private:
     /// Alpaka Queue
-    traccc::alpaka::queue m_queue;
+    std::shared_ptr<traccc::alpaka::queue> m_queue;
     /// Alpaka Vecmem objects, to get the memory resources
-    traccc::alpaka::vecmem_objects m_vecmem_objects;
+    std::shared_ptr<traccc::alpaka::vecmem_objects> m_vecmem_objects;
 
     /// Host memory resource
     ::vecmem::memory_resource& m_host_mr;
```

Comment:
Since we have the bare references, move-assignment is actually disabled at the moment. :frowning: Fine, let's not change that in this PR, but then why do we need to even make it explicit? The code was already implicitly not allowing any assignments anyway. :thinking: